design_notes/XDN03 - xlang Type System.md

```diff
@@ -206,12 +206,19 @@ Pre-existing members may not be removed or modified in any way.
 
 ### Events
 
-### Intrinsic Members
+### Intrinsic Operations
 
-All interfaces and runtime classes support the following members.
+All interfaces and runtime classes support the following operations intrinsically. The projection of
+these intrinsic operations is language specific. For languages that have a standard projection for
+a given intrinsic operation (such as C++ ToString or JavaScript equals), the projection is free to
+project the operation in an appropriate manner, if at all.
 
 #### ToString
 
+Returns a string that represents the current object. Typically, this defaults to the object's fully
+namespace-qualified type name, but can be customized. For example, an xlang type that represents a
+[JSON value](http://json.org/) would return the text representation of the JSON value from ToString.
+
 Projection language examples:
 
 - C++ N/A
```

Comment:
Also need Python and C#. 

I don't think we need Kotlin and Swift due to how those interop w/ Java and ObjC respectively. 

TS vs. JS is interesting since a TS projection of xlang is probably more "natural and familiar" than a JS one. However, I don't think we can make TS a requirement, so it seems reasonable to add code examples in both TS and JS

Reply:
I've added a philosophy section with links to all these languages